SeaBattle/BaseShip.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;

```

Comment:
You have a lot of usings here, which are not used, clear them.

Reply:
Cleared.